manifest.webapp

```diff
@@ -80,8 +80,12 @@
       ]
     },
     "settings": {
-      "description": "Required by the cozy-bar to display Claudy and know which applications are coming soon",
-      "type": "io.cozy.settings",
+      "description": "Required by the push banners and the cozy-bar to display Claudy and know which applications are coming soon",
+      "type": "io.cozy.settings"
+    },
+    "oAuthClients": {
```

Comment:
@Crash-- Do you remember what was the problem on Flagship app related to permission changes? Do we had a bug with adding permissions? Or with removing permissions?

Reply:
Yep I do. 

We have issue if we have a local version of the app. In that case: 

A/ if the local version is newer than the one installed on the cozy, then we will have issue if we have added new permissions 
B/ if the local version is older than the one installed on the cozy then we will have issue if we have removed permissions

Since the store app can't be in the A/ case, we don't have any issue when we add permissions. ATM, only the home app is concerned by this issue. 

We can have the issue on B/ for all the apps. So for now, we should avoid to remove permission ;)

Reply:
Je comprends qu'on est bon, ici je ne fais que rajouter des permissions.